app/forms/flood_risk_engine/steps/user_type_form.rb

```diff
@@ -21,7 +21,13 @@ def params_key
       # when we save the enrollment
       def save
         model.org_type = org_type.to_sym
+        super
+        # Need to go back and then forward as organisation change will effect
+        # onward journey. Without the back and forth a user cannot go back
+        # through the user type selection.
+        enrollment.go_back
```

Comment:
I agree that this isn't perfect, but was the simplest way I could get the backward functionality to work. This back and forth should only be needed if a attribute change alters which work flow is being used. So at the moment - only when the organisation type changes.

BTW, `super` is saving the `organisation` object and not the enrollment.